cranelift/zkasm_data/generated/_should_fail_unreachable.zkasm

```diff
@@ -0,0 +1,9 @@
+start:
+  zkPC + 2 => RR
+  :JMP(function_1)
+  :JMP(finalizeExecution)
+function_1:
+  UNREACHABLE: unreachable
```

Comment:
Producing an invalid instruction is a good way to go about it, sure. Another way I’ve seen it being done is to produce an infinite loop (which we already pretty much have at `finalizeExecution`)